### PR TITLE
Regular scenario roles

### PR DESCRIPTION
**Pull request by egorka-sereda**
regular scenario roles

* add aggregator role support to regular scenario
* add proposer role support to regular scenario
* add sync committee role support to regular scenario
* add sync committee contribution role support to regular scenario
* rename regular.go -to> regular_attester.go
* fix createDuty() func. now it works also for sync committee roles
* other minor fixes

### CODE REVIEW

File: integration/qbft/scenarios/integration_test_type.go [hunk 1]
```diff
@@ -99,7 +96,7 @@ func (it *IntegrationTest) bootstrap(ctx context.Context) (*scenarioContext, err
 
 	forkVersion := protocolforks.GenesisForkVersion
 
-	ln, err := p2pv1.CreateAndStartLocalNet(ctx, loggerFactory, forkVersion, len(it.OperatorIDs), len(it.OperatorIDs)-1, false)
+	ln, err := p2pv1.CreateAndStartLocalNet(ctx, loggerFactory, forkVersion, len(it.OperatorIDs), len(it.OperatorIDs)/2, false)
```
Comment (nkryuchkov) [inline integration/qbft/scenarios/integration_test_type.go:99]: Could you please change it to `len(it.OperatorIDs)-1`? It was changed on `integration-tests` before merging to increase stability of integration tests in the CI

File: integration/qbft/scenarios/integration_test_type.go [hunk 2]
```diff
@@ -280,12 +257,6 @@ func (it *IntegrationTest) Run() error {
 					return fmt.Errorf("stored instance is nil, operator ID %v, instance index %v", operatorID, i)
 				}
 
-				jsonInstance, err := json.Marshal(storedInstance)
```
Comment (nkryuchkov) [inline integration/qbft/scenarios/integration_test_type.go:283]: Could you please keep it? It was also recently added to `integration-tests`. It's a debug log but I'm not sure if integration tests are stable enough in the CI, so if CI fails, this log may be useful. By the way, we probably need to add a to-do to remove it later after we make sure that integration tests are stable in the CI

File: integration/qbft/scenarios/regular_aggregator.go [hunk 3]
```diff
@@ -82,14 +69,14 @@ func regularInstanceValidator(consensusData *spectypes.ConsensusData, operatorID
 			Data: encodedConsensusData,
 		}).Encode()
 		if err != nil {
-			return fmt.Errorf("encode prepare data: %w", err)
+			panic(err)
```
Comment (nkryuchkov) [inline integration/qbft/scenarios/regular_aggregator.go:72]: https://github.com/bloxapp/ssv/pull/793#discussion_r1067178852 (you can just pull from stage)

File: integration/qbft/scenarios/regular_attester.go [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+		}).Encode()
+		if err != nil {
+			return fmt.Errorf("encode proposal data: %w", err)
+		}
+
+		prepareData, err := (&specqbft.PrepareData{
+			Data: encodedConsensusData,
+		}).Encode()
+		if err != nil {
+			panic(err)
```
Comment (nkryuchkov) [inline integration/qbft/scenarios/regular_attester.go:72]: https://github.com/bloxapp/ssv/pull/793#discussion_r1067178852

File: integration/qbft/scenarios/round_change.go [hunk 5]
```diff
@@ -103,14 +103,14 @@ func roundChangeInstanceValidator(consensusData []byte, operatorID spectypes.Ope
 			Data: consensusData,
 		}).Encode()
 		if err != nil {
-			return fmt.Errorf("encode prepare data: %w", err)
+			panic(err)
```
Comment (nkryuchkov) [inline integration/qbft/scenarios/round_change.go:106]: https://github.com/bloxapp/ssv/pull/793#discussion_r1067178852

File: network/p2p/test_utils.go [hunk 6]
```diff
@@ -83,7 +83,7 @@ func CreateAndStartLocalNet(pctx context.Context, loggerFactory LoggerFactory, f
 			}
 			go func(node network.P2PNetwork, logger *zap.Logger) {
 				defer wg.Done()
-				ctx, cancel := context.WithTimeout(pctx, time.Second*15)
+				ctx, cancel := context.WithTimeout(pctx, time.Second*10)
```
Comment (nkryuchkov) [inline network/p2p/test_utils.go:86]: Please pull recent changes from from stage

File: integration/qbft/scenarios/integration_test_type.go (hunk 2 above)
Comment (egorka-sereda) [reply integration/qbft/scenarios/integration_test_type.go:283]: what do you mean? could you `suggest change` please?

File: integration/qbft/scenarios/regular_aggregator.go (hunk 3 above)
Comment (egorka-sereda) [reply integration/qbft/scenarios/regular_aggregator.go:72]: it overwrites on future brunches

File: integration/qbft/scenarios/round_change.go (hunk 5 above)
Comment (egorka-sereda) [reply integration/qbft/scenarios/round_change.go:106]: it overwrites on future brunches

File: integration/qbft/scenarios/regular_attester.go (hunk 4 above)
Comment (egorka-sereda) [reply integration/qbft/scenarios/regular_attester.go:72]: it overwrites on future brunches

**Review by nkryuchkov — APPROVED** [review]

**Pull request merged**